### ISSUE TITLE
Choose Icons

### ISSUE THREAD

**Issue by jrvaja** [contentco/quill-emoji#39]
@olsgreen @cbracco @himynameistimli @klaslundberg @vloginov 
Hey Team,

Is there any way to choose the list of emojis? 
So It will fetch and show only listed emojis not all.

Regards,

**Comment by olsgreen** [comment]
No, not that I've seen.

**Comment by himynameistimli** [comment]
Yeah right now we just use a single sprite map for all of the emojis, though it is a bit out of date. I understand from a loading / package-size perspective but what are the practical / user implications of having dynamic emoji fetching?
